RecoTracker/IterativeTracking/python/MixedTripletStep_cff.py

```diff
@@ -31,12 +31,13 @@
 # SEEDING LAYERS
 from RecoLocalTracker.SiStripClusterizer.SiStripClusterChargeCut_cfi import *
 from RecoTracker.IterativeTracking.DetachedTripletStep_cff import detachedTripletStepSeedLayers
-mixedTripletStepSeedLayersA = cms.EDProducer('SeedingLayersEDProducer',
-     layerList = cms.vstring('BPix2+FPix1_pos+FPix2_pos', 'BPix2+FPix1_neg+FPix2_neg'),
-#    layerList = cms.vstring('BPix1+BPix2+BPix3', 
+import RecoTracker.TkSeedingLayers.seedingLayersEDProducer_cfi as _seedmod
+mixedTripletStepSeedLayersA = _seedmod.seedingLayersEDProducer.clone(
+     layerList = ['BPix2+FPix1_pos+FPix2_pos', 'BPix2+FPix1_neg+FPix2_neg'],
+#    layerList = ['BPix1+BPix2+BPix3', 
 #        'BPix1+BPix2+FPix1_pos', 'BPix1+BPix2+FPix1_neg', 
 #        'BPix1+FPix1_pos+FPix2_pos', 'BPix1+FPix1_neg+FPix2_neg', 
-#        'BPix2+FPix1_pos+FPix2_pos', 'BPix2+FPix1_neg+FPix2_neg'),
+#        'BPix2+FPix1_pos+FPix2_pos', 'BPix2+FPix1_neg+FPix2_neg'],
     BPix = cms.PSet(
```

Comment:
the implementation in cfipython is
```python
seedingLayersEDProducer = cms.EDProducer('SeedingLayersEDProducer',
  layerList = cms.vstring(),
  BPix = cms.PSet(),
  FPix = cms.PSet(),
  TIB = cms.PSet(),
  TID = cms.PSet(),
  TOB = cms.PSet(),
  TEC = cms.PSet(),
  MTIB = cms.PSet(),
  MTID = cms.PSet(),
  MTOB = cms.PSet(),
  MTEC = cms.PSet(),
```
So, I expect that the following should work (same for all other PSet in this module as well as in `mixedTripletStepSeedLayersB` in this file.
```suggestion
    BPix = dict(
```